### DISCUSSION TITLE
Is catppuccin lualine integration supported?

### DISCUSSION THREAD

**Discussion by jthro** [nix-community/nixvim#1856] [General]
Just asking because from what I can tell the answer is no, and I'm interested in making an issue adding it it.

**Comment by MattSturgeon** [comment]
Most of our colorscheme plugins integrate to some extent with related plugins, IIRC.

Can't see any special magic in catppuccin though: https://github.com/nix-community/nixvim/blob/53e85da2409ddfeef27e8cdc9c020433614329f9/plugins/colorschemes/catppuccin.nix#L319

_Usually_ calling `:colorscheme` early enough in init.lua is enough, though (which all of our colorscheme plugins do).

**Reply by GaetanLepage** [reply]
@jthro I think that you can indeed give it a shot in a PR.
Don't hesistate to `grep` for `lualine` in the `plugins/by-name` folder and see how it is done for the other colorschemes.
